docs/template_packs.rst

```diff
@@ -4,7 +4,7 @@
 How to create your own template packs
 =====================================
 
-First you will have to name your template pack, for this you can't use the name of one of the available template packs in crispy-forms, due to name collisions. For example, let's say in the company we work for, a designer has come up with a CSS bootstrap internally known as ``chocolate``. The company has a Django project that needs to start using ``chocolate``, therefore we need to create a folder named ``chocolate`` within our templates directory. Check your ``TEMPLATE_DIRS`` setting in Django and pick your preferred path.
+First you will have to name your template pack, for this you can't use the name of one of the available template packs in crispy-forms, due to name collisions. For example, let's say in the company we work for, a designer has come up with a CSS bootstrap internally known as ``chocolate``. The company has a Django project that needs to start using ``chocolate``, therefore we need to create a folder named ``chocolate`` within our templates directory. Check your ``TEMPLATE`` setting in Django and pick your preferred path. You can check it in the Django shell with `from django.conf import settings; settings.TEMPLATES`.
```

Comment:
Thanks f-r this. As per your note on the PR it is ``TEMPLATES`` not ``TEMPLATE``. Template is confusing as it's still a setting but relates to postgres.

Do you still have time to fix this typo?

It may even be worth adding a link to the Django docs?

Templates (what we need) - https://docs.djangoproject.com/en/3.0/ref/settings/#templates
Template (a postgres setting) - https://docs.djangoproject.com/en/3.0/ref/settings/#template

Reply:
I don't remember what this is about. Feel free to close